SR2023/2022-10-17-place-rejection.md

```diff
@@ -0,0 +1,17 @@
+---
+to: Student Robotics 2023 rejections
+subject: Student Robotics 2023 is sadly full
+---
+
+Hello!
+
+I'm afraid Student Robotics 2023 is full, and we are unable to offer you a place. We've had more schools interested than we have places for.
```

Comment:
Should we apologies for not getting back to them sooner? It's been over two weeks since the sign-up deadline.